packages/backend-plugin-api/src/services/definitions/RootHealthService.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @public
+ */
+export interface RootHealthService {
+  /**
+   * Get the liveness status of the backend.
+   */
+  getLiveness(): Promise<{ status: number; payload?: any }>;
```

Comment:
yeah again, making these payloads `JsonObject` conveys the limitation that they are being serialized as a response body.